build.sbt

```diff
@@ -17,7 +17,8 @@ val dependencies = Seq(
   "uk.gov.hmrc" %% "play-whitelist-filter" % "2.0.0",
   "com.github.kxbmap" %% "configs" % "0.4.4",
   "org.typelevel" %% "cats-core" % "1.1.0",
-  "com.google.guava" % "guava" % "23.0"
+  "com.google.guava" % "guava" % "23.0",
+  "uk.gov.hmrc" % "totp-generator_2.11" % "0.5.0"
```

Comment:
you don't need the `_2.11`, you can do
```
"uk.gov.hmrc" % "totp-generator" %% "0.5.0"
```
with two `%`'s instead to automatically derive the scala version

Reply:
thanks